config/defaults.go

```diff
@@ -0,0 +1,95 @@
+/*
+   Copyright The Soci Snapshotter Authors.
+
+   Licensed under the Apache License, Version 2.0 (the "License");
+   you may not use this file except in compliance with the License.
+   You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.
+*/
+
+/*
+   Copyright The containerd Authors.
+
+   Licensed under the Apache License, Version 2.0 (the "License");
+   you may not use this file except in compliance with the License.
+   You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.
+*/
+
+/*
+   Copyright 2019 The Go Authors. All rights reserved.
+   Use of this source code is governed by a BSD-style
+   license that can be found in the NOTICE.md file.
+*/
+
+package config
+
+// Config (root) defaults
+const (
+	defaultMetricsNetwork = "tcp"
+)
+
+// ServiceConfig defaults
+const (
+	defaultImageServiceAddress = "/run/containerd/containerd.sock"
+)
+
+// FSConfig defaults
+const (
+	defaultFuseTimeoutSec = 1
+	// Amount of time the background fetcher will wait once a new layer comes in
```

Comment:
nit: I think it's useful to use the doc comment format (the first word is the name of the identifier) even for unexported constants since it makes it easier to export them later.

Reply:
Agreed. I'll fix it, this was mainly copied over from `fs.go`.